grafast/grafserv/src/options.ts

```diff
@@ -43,6 +43,7 @@ export function defaultMaskError(
     const hash = sha1(String(error));
     const errorId = randomString();
     console.error(
+      "%s\n%O",
       `Masked GraphQL error (hash: '${hash}', id: '${errorId}')`,
       error,
```

Comment:
The error being logged was actually the wrong error; it should have been the `.originalError` (which is unlikely to have beautiful formatting) - this is the actual useful underlying error that will help you fix the bug, rather than just telling you where in the GraphQL query it occurred (information that's already available via the client).

I'm not convinced that logging the pretty GraphQL error is all that useful, but I've left it in anyway. If enough people need support because they haven't read through the "pretty" error to the actual significant part of the error I'll remove it in a future update.

```suggestion
      "Masked GraphQL error (hash: '%s', id: '%s')\n%s\n%O",
      hash,
      errorId,
      error,
      error.originalError ?? error,
    )
```